walker/walk_creator.py

```diff
@@ -37,9 +39,38 @@ def serialize_walker(self):
             if m.type == "distance":
                 muscles.append([self.joint_index(m.j0), self.joint_index(m.j1), {"type": m.type}])
             elif m.type == "muscle":
-                muscles.append([self.joint_index(m.j0), self.joint_index(m.j1), {"type": m.type, "amplitude": m.amplitude, "phase": m.phase}])
+                muscles.append([self.joint_index(m.j0), self.joint_index(m.j1),
+                                {"type": m.type, "amplitude": m.amplitude, "phase": m.phase}])
         return {"joints": joints, "muscles": muscles}
 
+    def serialize_walker_sodarace(self):
+        walker_dict = {
+            "useLEO": True,
+            "nodes": [
+            ],
+            "connections": [
+            ],
+        }
+        for j in self.joints:
+            walker_dict["nodes"].append({
+                "x": j.x,
+                "y": j.y,
+            })
+        for m in self.muscles:
+            if m.type == "distance":
+                walker_dict['connections'].append({
+                    "sourceID": str(self.joint_index(m.j0)),
+                    "targetID": str(self.joint_index(m.j1)),
+                    "cppnOutputs": [0, 0, 0, -10.0]
+                })
+            elif m.type == "muscle":
+                walker_dict['connections'].append({
+                    "sourceID": str(self.joint_index(m.j0)),
+                    "targetID": str(self.joint_index(m.j1)),
+                    "cppnOutputs": [0, 0, m.phase, m.amplitude]
```

Comment:
@dmahan93 Just making sure this makes sense to construct a dict like the example ones you provided. The code does not use the first two indexes of `cppnOutputs`, right, assuming `useLEO=True`? Also do you know the purpose of the `useLEO` option?

Reply:
@herbiebradley That's correct yeah. The useLEO option is a hyperneat thing, (pg. 3 http://eplex.cs.ucf.edu/papers/szerlip_ecal13.pdf), so we can safely ignore it.